# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/btscanner/dontchecklibsdp.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN btscanner-1.0/configure.in btscanner-1.0-new/configure.in
--- btscanner-1.0/configure.in	2003-10-14 13:01:20.000000000 +0200
+++ btscanner-1.0-new/configure.in	2004-12-29 10:30:06.770321632 +0100
@@ -12,7 +12,7 @@
 # Checks for libraries.
 AC_HAVE_LIBRARY(curses,,[AC_MSG_ERROR([Curses not found])])
 AC_HAVE_LIBRARY(bluetooth,,[AC_MSG_ERROR([Bluetooth not found])])
-AC_HAVE_LIBRARY(sdp,,[AC_MSG_ERROR([SDP not found])])
+# AC_HAVE_LIBRARY(sdp,,[AC_MSG_ERROR([SDP not found])])
 AC_HAVE_LIBRARY(pthread,,[AC_MSG_ERROR([Pthreads not found])])
 AC_HAVE_LIBRARY(gdbm,,[AC_MSG_ERROR([GDBM not found])])
 
